src/main/java/mate.academy.internetshop/controller/ItemController.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.internetshop.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Inject;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import org.apache.log4j.Logger;
+
+@WebServlet("/items")
+public class ItemController extends HttpServlet {
+    @Inject
+    private static ItemService itemService;
+    @Inject
+    private static BucketService bucketService;
+
+    private static final Long TEMP_BUCKET_ID = 0L;
+
+    private static final Logger log = Logger.getLogger(ItemController.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.setAttribute("items", itemService.getAll());
+        request.getRequestDispatcher("WEB-INF/views/items.jsp").forward(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String button = request.getParameter("+");
```

Comment:
"+"? button?
try to create more understandable name of the variables
the same in all controllers

Reply:
Renamed button to "Add"